### ISSUE TITLE
Missing obj in assets

### ISSUE THREAD

**Issue by haoyangzhanglab** [Kami-code/dexart-release#1]
Hello, congratulations on your excellent work. I am trying to run the code, but when I followed the steps in readme.md, I encountered the following bug. It seems that the assets compressed package does not contain all the object files, can you please update the file? In addition, is it convenient to give me some details about the hardware and experiment configuration of the training policy? I did not find a description in the paper. Thank you. Best wishes.
![image](https://github.com/Kami-code/dexart-release/assets/131220673/44205f54-743e-49c8-9c73-e837cd6c7d80)
![image](https://github.com/Kami-code/dexart-release/assets/131220673/c25f5a26-5d04-4d8a-8124-e295b2221346)

**Comment by Kami-code** [comment]
Hi. Thanks for using our benchmark. The problem you described is a known bug in laptop assets. You can ignore the warning because it will not affect your environment simulation and training. Usually, each training seed takes about 48 hours for Faucet, 72 h. for Bucket, 15 h. for Laptop, and 15 h. for Toilet on an NVIDIA RTX 2080Ti GPU when you use 
`--n 100 --workers 10 --iter 5000 --bs 500` in our training example.

**Comment by vlongle** [comment]
I'm running the example training script (laptop task) and getting fps=30 on an NVIDIA RTX 3090 GPU.  This means that it would take 92h and not 15h to train this task for me.  Do you have any idea why there might be such a discrepancy? 
![Screenshot 2023-06-28 at 6 45 43 PM](https://github.com/Kami-code/dexart-release/assets/37346413/3016a669-3a3e-47c0-99a3-927d774028bb)

**Comment by Kami-code** [comment]
Hi @vlongle . Could you provide the whole command you used for training?

**Comment by vlongle** [comment]
I'm using the command from the readme.
```
python examples/train.py --n 100 --workers 10 --iter 5000 --lr 0.0001 --seed 100 --bs 500 --task_name laptop --extractor_name smallpn --pretrain_path ./assets/vision_pretrain/laptop_smallpn_fulldata.pth
```

**Comment by Kami-code** [comment]
I inspected our training log, the parameters are the same. But your rollout time is suprisingly high while ours is ~30 and fps is 500. 
<img width="446" alt="20230628205049" src="https://github.com/Kami-code/dexart-release/assets/58084462/863434b9-761d-4940-afc6-fc535b40fe8e">
When the agent is collecting rollouts, the simulation speed depends on your CPU, memory and the number of workers specified. I am not sure if this is causing the difference. Since all our experiments on conducted on the cluster, I can provide you our resource requests in our config file.

> limits:
>   cpu: "12"
>   memory: "100Gi"
>   nvidia.com/gpu: "1"

**Comment by HXY-95** [comment]
> I inspected our training log, the parameters are the same. But your rollout time is suprisingly high while ours is ~30 and fps is 500. <img alt="20230628205049" width="446" src="https://private-user-images.githubusercontent.com/58084462/249471098-863434b9-761d-4940-afc6-fc535b40fe8e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg3NzUyMzUsIm5iZiI6MTcwODc3NDkzNSwicGF0aCI6Ii81ODA4NDQ2Mi8yNDk0NzEwOTgtODYzNDM0YjktNzYxZC00OTQwLWFmYzYtZmM1MzViNDBmZThlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjI0VDExNDIxNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE2OTMzODBlMTI0Nzg4NjM0YTIzYjkzNzBmYzU4NGNlMTU1NDMzYzdmNGJkOThlYTVmYmJlZGU4NjJlODM4YmQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.dNZuFd_-NypnZXhcFOwJXaCVdNs0Pua4aFGProsSVho"> When the agent is collecting rollouts, the simulation speed depends on your CPU, memory and the number of workers specified. I am not sure if this is causing the difference. Since all our experiments on conducted on the cluster, I can provide you our resource requests in our config file.
> 
> > limits:
> > cpu: "12"
> > memory: "100Gi"
> > nvidia.com/gpu: "1"

According to this setting, the training time of the laptop should be ((30+6)*5000)/3600=50 hours. However, you said the time is 15 hours.
